src/modules/ekf_att_pos_estimator/ekf_att_pos_estimator_main.cpp

```diff
@@ -1195,7 +1225,11 @@ FixedwingEstimator::task_main()
 					// Set up height correctly
 					orb_copy(ORB_ID(sensor_baro0), _baro_sub, &_baro);
 					_baro_ref_offset = _ekf->states[9]; // this should become zero in the local frame
```

Comment:
What should be the value of states[9] when this is called? It looks like this is the only assignment to _baro_ref_offset.